app/js/controllers/EditCtrl.js

```diff
@@ -334,18 +336,31 @@ KMCMenu.controller('EditCtrl', ['$scope','$http', '$timeout','PlayerData','Playe
 				delete $scope.playerData.config.cast;
 				delete $scope.playerData.externals[plugin.componentName];
 			}
-		} else {
+		} else if (plugin.label === "Receiver") {
 			if (willActive) {
 				$scope.playerData.externals[plugin.componentName] = {
 					active: true
 				};
 				senderPlugin.enabled = false;
+				airPlayPlugin.enabled = false;
 				senderElement.classList.add('disabled');
+				airPlayElement.classList.add('disabled');
 				delete $scope.playerData.config.cast;
-                delete $scope.playerData.externals[senderPlugin.componentName];
+				delete $scope.playerData.config.plugins.airplay;
+				delete $scope.playerData.externals[senderPlugin.componentName];
+				delete $scope.playerData.externals[airPlayPlugin.componentName];
 			} else {
 				senderElement.classList.remove('disabled');
-                delete $scope.playerData.externals[plugin.componentName];
+				airPlayElement.classList.remove('disabled');
+				delete $scope.playerData.externals[plugin.componentName];
+			}
+		} else if (plugin.label === "AirPlay") {
+			if (willActive) {
+				airPlayPlugin.enabled = true;
```

Comment:
need to disable the cast receiver as you do for cast sender

Reply:
AirPlay isn't related to the cast except the fact it's under the same panel.

Reply:
so why you disabled air play when the receiver is checked?

Reply:
You don't want to bundle airplay if it's a receiver player.